src/models/StyledComponent.js

```diff
@@ -252,6 +252,9 @@ class BaseStyledComponent extends Component<*, BaseState> {
   }
 }
 
+BaseStyledComponent.isReactComponent =
```

Comment:
Why does this need to be a static property instead of an inherited one? I thought the recommended way to check for this stuff now was `react-is` also..

Reply:
Hey thanks for the comment!

I'm not sure I see another way of making it "inherited", but I'm all ears. What do you propose?

Didn't know about react-is, but I'll run another test to see if it correctly identifies the prototype of a styled-component as a React component.